Sources/TuistLoader/Loaders/RecursiveManifestLoader.swift

```diff
@@ -65,8 +65,12 @@ public class RecursiveManifestLoader: RecursiveManifestLoading {
         var paths = Set(paths)
         while !paths.isEmpty {
             paths.subtract(cache.keys)
-            let projects = try Array(paths).map(context: ExecutionContext.concurrent) {
-                try manifestLoader.loadProject(at: $0)
+            let projects = try Array(paths).map(context: ExecutionContext.concurrent) { path -> ProjectDescription.Project in
+                if path.extension == "json" {
+                    return try JSONDecoder().decode(ProjectDescription.Project.self, from: try fileHandler.readFile(path))
+                } else {
+                    return try manifestLoader.loadProject(at: path)
+                }
```

Comment:
Do you think it's fine if for SPM packages we generate just the json serialized `Project.swift`, and here we decode if from JSON instead of loading the swift manifest?